README.md

```diff
@@ -35,14 +35,16 @@ Then, install Python Cozy Management :
 * **get_oom_scores**: get the Out Of Memory score for each application;
 * **rebuild_app <app>**: reinstall every npm dependencies of an application;
 * **rebuild_all_apps**: reinstall every npm dependencies of all applications (useful when upgrading Node version);
-* **migrate_2_node4**: 
-* **install_requirements**: 
+* **migrate_2_node4**:
+* **install_requirements**:
 * **install_cozy**: install the Cozy server;
 * **wait_couchdb**: wait until couchdb has started;
 * **wait_cozy_stack**: wait until full Cozy stack has started;
 * **emulate_smtp [--bind <ip>] [--port <port>]** starts a fake SMTP server on port 25 for debugging purpose;
 * **backup**: create a timestamped backup of Cozy configuration and data into `/var/lib/cozy/backups`;
 * **restore <backup_filename>**: restore a backup;
+* **install_weboob**: install weboob from devel repository
+* **update_weboob_modules**: update weboob modules
```

Comment:
Cool. Maybe add a hint in the documentation that `install_weboob` also works for updates? (or create a dummy `update_weboob` that calls `install_weboob` :))